demos/python_demos/formula_recognition_demo/formula_recognition_demo.py

```diff
@@ -1,13 +1,10 @@
 #!/usr/bin/env python3
 """
  Copyright (c) 2020 Intel Corporation
-
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
-
       http://www.apache.org/licenses/LICENSE-2.0
-
```

Comment:
Why? :)

Reply:
Accidently deleted this, returned empty lines